pkg/config/config.go

```diff
@@ -26,6 +26,7 @@ var Flags = &Config{}
 var dnsZonesString string
 
 func init() {
+	flag.Var(&Flags.DefaultController, "default-controller", "kind of default controller to use. should be one of 'standard', 'public', 'private', or 'off'.")
```

Comment:
PRD only has 3 options of  external/internal/none, what is the standard option?

Reply:
the current live experience of not reconciling annotations. It makes it so we can decide when we want to opt into this without having to cut a new operator release (and can roll back if needed).

Reply:
Without standard we wouldn't be able to get this in until we had the new api experience ready and we wouldn't be able to bring the new api experience to older versions of the operator

Reply:
add the description of these options into the PR description with the implications of public/private overwriting changes. otherwise looks good!